repos/shared-library/src/Utility/OmekaValue.php

```diff
@@ -20,6 +20,12 @@ public static function translateValue($representation, $term, $lang = '')
         if (!$representation instanceof ItemRepresentation && !$representation instanceof ItemSetRepresentation) {
             throw new LogicException('Only Items and ItemSets can be translated.');
         }
+
+        // Examples of input/output:
+        // en-gb -> en
+        // en-us -> en
+        // en    -> en
+        // cy-gb -> cy
         $primaryLang = explode('-', $lang)[0] ?? $lang;
```

Comment:
We should use `Locale::parseLocale()` for this

Reply:
Changed that now to use both the parse and comparisons